sdk/cosmosdb/cosmos/src/request/request.ts

```diff
@@ -128,8 +129,15 @@ export async function getHeaders({
   }
 
   if (options.maxIntegratedCacheStalenessInMs && resourceType === ResourceType.item) {
-    headers[Constants.HttpHeaders.DedicatedGatewayPerRequestCacheStaleness] =
-      options.maxIntegratedCacheStalenessInMs;
+    if (typeof options.maxIntegratedCacheStalenessInMs === "number") {
+      headers[Constants.HttpHeaders.DedicatedGatewayPerRequestCacheStaleness] =
+        options.maxIntegratedCacheStalenessInMs.toString();
+    } else {
+      logger.error(
+        `RangeError: maxIntegratedCacheStalenessInMs "${options.maxIntegratedCacheStalenessInMs}" is not a valid parameter.`
```

Comment:
Should we call out 'parameter value'  instead of parameter.

Reply:
${options.maxIntegratedCacheStalenessInMs} is the value.